itests/eth_config_test.go

```diff
@@ -0,0 +1,64 @@
+// stm: #integration
+package itests
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/chain/types/ethtypes"
+	"github.com/filecoin-project/lotus/itests/kit"
+)
+
+func TestEthFilterAPIDisabledViaConfig(t *testing.T) {
+	ctx := context.Background()
+
+	kit.QuietMiningLogs()
+
+	// don't pass kit.RealTimeFilterAPI() so ActorEvent.EnableRealTimeFilterAPI is false
```

Comment:
Could you update this comment to reflect that the flag has be changed to `Fevm.Events.DisableRealTimeFilterAPI`